.golangci.yml

```diff
@@ -24,12 +24,22 @@ linters:
     - gosec
 
 issues:
+  exclude:
+    - "builder.Module undefined"
+    - "builder.Component undefined"
+    - "builder.Metrics undefined"
+    - "builder.ComplianceConfig undefined"
+    - "builder.BaseFlags undefined"
+    - "builder.BuildConsensusFollower"
+    - "StakedAccessNodeBuilder has no field or method"
+    - "FlowAccessNodeBuilder has no field or method"
+    - "ObserverNodeBuilder has no field or method"
+    # Disable false positives
```

Comment:
this doesn't seem safe for the rest of `flow-go`

Reply:
Removed. Reiterating, it was meant to cover an observer in a different repository by an instruction weeks ago.